README.md

```diff
@@ -69,6 +69,18 @@ This package assumes usage within a Qubit experience. If using elsewhere you can
 
 The Qubit Recommendations API endpoint.
 
+### currency
+*type*: `String`
+*default*: `currency` emitted in QP view event
+
+Allows you to override the currency for the recs endpoint locale; e.g. 'GBP'
```

Comment:
This technically isn't an override, it's a default because in the code you fall back to it (instead of prioritising it above the value from UV).

Could we rename these options to defaultCurrency and defaultLanguage? And all the way through the code

Reply:
Ah, yeah you're right. In which case, I would actively change this so it is defaultCurrency and defaultLanguage. Reason being that I think using the event value when it is available is better than hardcoding it for all users.